src/responsibleai/conda_envs/python-aml-rai.yaml

```diff
@@ -13,5 +13,6 @@ dependencies:
     - azureml-mlflow
     - responsibleai~=0.17.0
     - raiwidgets~=0.17.0
+    - markupsafe<=2.0.1
```

Comment:
Can you put a brief note here as well as to why we're upper-bounding this. It's possible that this could wind up detached from the PR, and hence the context lost. Just say it's a workaround, and give a link to the same issue as the PR description.

Reply:
Thanks for the comment! Note added